relay-event-normalization/src/normalize/span/description/sql/parser.rs

```diff
@@ -344,3 +381,62 @@ impl Dialect for DialectWithParameters {
         self.0.parse_statement(parser)
     }
 }
+
+#[cfg(test)]
+mod tests {
+
+    use sqlparser::parser::Parser;
+
+    use super::*;
+
+    #[test]
+    fn depth() {
+        let query = "SELECT (a + b + (c1 + (c2 + c3)) + c)";
+        let mut parsed = Parser::parse_sql(&GenericDialect {}, query).unwrap();
+        let mut visitor = NormalizeVisitor::new();
+        parsed.visit(&mut visitor);
+        assert_eq!(visitor.max_depth, 8);
```

Comment:
Both the parentheses and the expression within account for one level.

Reply:
I kind of loas why `8` here.

Reply:
The parser creates a new `Expression` object for both the parentheses _and_ every binary operation contained within the parentheses.

Reply:
cool, thanks! 
That's where my confusion came from.